thirdparty/LICENSE

```diff
@@ -0,0 +1,283 @@
+The SINGA project contains subcomponents with separate copyright
```

Comment:
Since there are multiple licenses in this file, you should separate them into different files. Each of those files includes an individual license.